introduction.md

```diff
@@ -1,2 +1,9 @@
 Hi this is my introduction. I go by the handle of - Rockncoder.
-My favorite code school path's are JavaScript, HTML & CSS, and iOS.
\ No newline at end of file
+My favorite code school path's are JavaScript, HTML & CSS, and iOS.
+My Skills
+* JavaScript
+* C++
+* C#
+* Objective-C
+* Java
+* Swift
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?